test/e2e/basic_test.go

```diff
@@ -50,15 +50,92 @@ func testCreateDefaultCluster(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = pravega_e2eutil.DeleteCluster(t, f, ctx, pravega)
+	err = pravega_e2eutil.DeleteCluster(t, f, pravega)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// No need to do cleanup since the cluster CR has already been deleted
 	doCleanup = false
 
-	err = pravega_e2eutil.WaitForClusterToTerminate(t, f, ctx, pravega)
+	err = pravega_e2eutil.WaitForClusterToTerminate(t, f, pravega)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+// Test restart Pravega cluster with the same name(issue 91)
+func testRestartDefaultCluster(t *testing.T) {
```

Comment:
I think the word "recreate" better describes this test case than "restart". IMO "restart" implies using the exact same cluster object, which is not the case.

Reply:
You are right